### PR TITLE
fix wierd burning cable

### PR DESCRIPTION
**Pull request by botn365**

### CODE REVIEW

File: src/main/java/gregtech/api/graphs/GenerateNodeMapPower.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import gregtech.api.metatileentity.MetaPipeEntity;
 import gregtech.api.metatileentity.implementations.GT_MetaPipeEntity_Cable;
 import ic2.api.energy.tile.IEnergySink;
+import net.minecraft.server.MinecraftServer;
```
Comment (Glease) [inline src/main/java/gregtech/api/graphs/GenerateNodeMapPower.java:14]: uh? a single import with no other changes?

Comment (Dream-Master) [reply src/main/java/gregtech/api/graphs/GenerateNodeMapPower.java:14]: check the other commits @Glease

Comment (Glease) [reply src/main/java/gregtech/api/graphs/GenerateNodeMapPower.java:14]: It's still an unused import. It'd be best if anyone can remove it.

Comment (Dream-Master) [reply src/main/java/gregtech/api/graphs/GenerateNodeMapPower.java:14]: @botn365

**Review by Glease — APPROVED** [review]

**Pull request merged**